pages/classes/[classNumber]/print.tsx

```diff
@@ -93,13 +86,13 @@ export const getStaticProps: GetStaticProps = async ({ locale, params }) => {
         "common",
         "classes",
       ])),
-      classItem,
-      classroomOverview,
+      classroom,
       studentList: studentList!.sort(
-        // Put Students with no class No. first
+        // Put Students with no class No. first.
         (a, b) => (a.class_no || 0) - (b.class_no || 0),
       ),
     },
+    revalidate: 3000,
```

Comment:
the feature takes too long to load, and update for the print.tsx

Reply:
In hindsight, 50 minutes is way too long. I’ll look into that!

As for it taking too long to load, I want to reiterate that **this is an issue with hosting and not us**. There is nothing we can do until we can pay for a better host.